package.json

```diff
@@ -19,6 +19,7 @@
     "bootstrap-select": "^1.13.18",
     "datatables.net-dt": "^1.10.20",
     "jquery": "^3.5.0",
+    "node-sass": "^4.14.1",
```

Comment:
We have sass now??

Reply:
Not intentionally! Not sure how that got there but I will remove it.